index.js

```diff
@@ -26,6 +25,20 @@ bot.on("error", err => {
     console.log(err);
 });
 
+bot.on("guildMemberAdd",(guild, member) =>{
+    let userName = member.username
+    let cocChannel = process.env.COC_CHANNEL_ID
```

Comment:
Since this is optional, it might be good to only send this message if this feature is configured? Maybe even not bind this event handler at all if `COC_CHANNEL_ID` isn't set.